### ISSUE TITLE
ion channel review

### ISSUE THREAD

**Issue by tgbugs** [SciCrunch/NIF-Ontology#80]
Is this a taxonomy? How should we model these. Need to do a review.

**Comment by cmungall** [comment]
surely you'd use GO for functional classification, and something like panther for the protein family?

**Comment by tgbugs** [comment]
Yes (I assume you are referring to the branch of GO that starts from [GO:0022803](http://www.informatics.jax.org/searches/GO.cgi?id=GO:0022803)). The other two areas of interest for classification would be their subunit composition (which is the direct mapping to genes) and the types of mathematical models used to characterize the channel physiology which includes ion specificity and channel kinetics (probably a reasonably close mapping to GO).

Right now in NIF it seems like we are conflating genes, proteins, and full channels, which would make it difficult to claim that what we have is really a taxonomy.

One other thing here is that for neuroscientists we usually also include receptors whose ionotropic members fall into the ligand gated ion channel branch. (renamed the issue accordingly)

**Comment by cmungall** [comment]
Good point about the mathematical models, I hadn't considered this axis before. cc @dosumis

**Comment by dosumis** [comment]
We definitely need to work on the ion channel branch of GO MF.  We need better ways to formally represent voltage gating and functional classifications e.g. rectifier; delayed rectifier.   I'm also still dissatisfied with they way we deal with ion specificity.    We wouldn't be interested in the "types of mathematical models used to characterize the channel physiology" except as evidence for functional classification/ion specificity etc.

**Comment by tgbugs** [comment]
Yes, the idea is that models could provide more parametrized evidence for certain functional classifications, not that we would want to delve into whether certain channels had been modelled using stochastic vs deterministic techniques or something like that. So the models vis-a-vis the biology not other models (if that makes sense).

**Comment by apdavison** [comment]
I just stumbled upon this discussion while searching for ontologies relating to ion channels. It seems as though this article might be of interest: https://dx.doi.org/10.1093%2Fdatabase%2Fbaw017

**Comment by tgbugs** [comment]
Thanks for the link. That gives good coverage on the electrical and kinetic side which we will need to connect with the protein side.
